docs/design/using-motion-office-addins.md

```diff
@@ -50,12 +50,12 @@ Consider the size of the HTML canvas (task pane, dialog box, or content add-in)
 
 |Do|Don't|
 |:-----|:-----|
-| Use [recommended motion durations](https://developer.microsoft.com/fabric#/styles/web/motion). | Don't use exaggerated animations. Avoid creating experiences that embellish and distract your customers.
+| Use [recommended motion durations](https://developer.microsoft.com/fluentui#/styles/web/motion). | Don't use exaggerated animations. Avoid creating experiences that embellish and distract your customers.
 | Follow [recommended easing curves](/windows/uwp/design/motion/timing-and-easing#easing-in-fluent-motion).  |Don't move elements in a jerky or disjointed manner. Avoid anticipations, bounces, rubberband, or other effects that emulate natural world physics.|
 
 ![GIF that shows tiles loading using a gentle fade in against a GIF that shows tiles loading with bounce](../images/add-in-motion-character.gif)
```

Comment:
I checked the alt-text in this file and think we should change them all. Suggest replace "against" with "compared to".

Reply:
Done. I used "beside" instead of "compared to"